src/components/Badge/Badge.cy.tsx

```diff
@@ -248,4 +249,16 @@ describe('Badge component', () => {
 
         cy.get(BADGE_BUTTON).invoke('attr', 'title').should('equal', `${BADGE_TEXT}_1_${BADGE_TEXT}_2_${BADGE_TEXT}`);
     });
+
+    it('should have padding between icon and label', () => {
+        cy.mount(<Badge icon={<IconDocumentText />}>{BADGE_TEXT}</Badge>);
+
+        cy.get(BADGE_ICON_ID).should('have.class', BADGE_ICON_PADDING_CLASS);
```

Comment:
```suggestion
        cy.get(BADGE_ICON_ID).should('have.css', 'padding-right', '1rem');
```
Line not tested but it gives the idea